doc/src/python.rst

```diff
@@ -70,80 +76,89 @@ Examples
        lmp.command("pair_style lj/cut ${cut}")   # LAMMPS commands
        lmp.command("pair_coeff * * 1.0 1.0")
        lmp.command("run 100")
-    """
+   """
+
+   python source funcdef.py
+
+   python source inline "from lammps import lammps"
+
 
 Description
 """""""""""
 
 Define a Python function or execute a previously defined function or
```

Comment:
The first paragraph here should explain the 2 modes and the motivation for the user to use them. Then
new paragraphs can explain the details of the 2 modes using the content that is currently in this long paragraph.